test/e2e/framework/ssh/ssh.go

```diff
@@ -155,6 +156,7 @@ func NodeSSHHosts(c clientset.Interface) ([]string, error) {
 	}
 	wg.Wait()
 
+	e2elog.Logf("Found working SSH hosts: %s", strings.Join(sshHosts, ", "))
```

Comment:
Looking into: https://github.com/kubernetes/kubernetes/pull/101866 - this clearly broke scalability correctness tests.

From this line in the log file:
```
ï¿½[1mSTEPï¿½[0m: SSH'ing to 0 nodes and running echo "Hello from $(whoami)@$(hostname)"
```

We know that after that PR, the function returns 0 nodes (where it used to return at least 1).

We should fix the `canConnect` function.